src/View/Shortcodes/EmbedShortcodeProvider.php

```diff
@@ -89,7 +89,24 @@ public static function handle_shortcode($arguments, $content, $parser, $shortcod
         }
 
         // Process embed
-        $embed = $embed->getEmbed();
+        try {
+            $embed = $embed->getEmbed();
+        }
+        catch(\Embed\Exceptions\InvalidUrlException $e) {
```

Comment:
95 | ERROR | [x] Expected 1 space(s) after CATCH keyword; 0 found
```suggestion
        catch (\Embed\Exceptions\InvalidUrlException $e) {
```

Reply:
Could you change this to 
`use \Embed\Exceptions\InvalidUrlException;` (top of the php file)
`catch (InvalidUrlException $e) {`